### PR TITLE
Some changes found during an upgrade...

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

File: c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl [hunk 1]
```diff
@@ -61,7 +61,7 @@ You should also add a Layer group named 'background' with the background layers.
     cp CONST_create_template/{{package}}/static-ngeo/less/mobile.less {{package}}/static-ngeo/less
 
 7. The checkers will be refactored, you should do at least the following
-   change in your vars_{{package}}.yaml:
+   change in your `vars_{{package}}.yaml` (be careful if you use legacy components):
```
Comment (ger-benjamin) [inline c2cgeoportal/scaffolds/update/CONST_CHANGELOG.txt_tmpl]: chnage -> chnages (with the 's')

File: c2cgeoportal/views/entry.py [hunk 2]
```diff
@@ -584,7 +584,9 @@ def _fill_internal_wms(self, l, layer, wms, wms_layers, errors):
         else:
             if self.default_ogc_server.type != OGCSERVER_TYPE_GEOSERVER:
                 errors.add(
-                    "The layer '{0!s}' is not defined in WMS capabilities".format(wmslayer)
+                    "The layer '{}' ({}) is not defined in WMS capabilities from '{}'".format(
+                        wmslayer, layer.name, self.default_ogc_server.name
```
Comment (ger-benjamin) [inline c2cgeoportal/views/entry.py]: I think that's egal for a human but `layer.layer` isn't more precise ?

And don't we do the same trick (layer.name -> layer.name) for wmts layers ?

**Comment by ger-benjamin** [comment]
Some comments but that's look good. Thanks :-)

**Comment by sbrunner** [comment]
Thanks @ger-benjamin :-)

File: c2cgeoportal/views/entry.py (hunk 2 above)
Comment (sbrunner) [inline c2cgeoportal/views/entry.py]: layer.layer is the same as `wmslayer` than it's displayed, and `layer.name` can be helpful when we have more than one layer with the same `layer`...

I don't understand, this is for layerv1, for layerwms and layerwmts it's working with layer(s) and name for a long time :-)

**Pull request merged**